File: Easy/mixed_content.py
data = ['8,33,21,0,16,50,37,0,melon,7,apricot,peach,pineapple,17,21', '24,13,14,43,41']

#import sys
#test_cases = open(sys.argv[1], 'r')
#data = test_cases.readlines()
for test in data:
	test = test.rstrip()
	test = test.split(",")
	words = []
	nums = []
	for i in test:
		try:
			n = int(i)
			nums.append(i)
		except:
			words.append(i)
	if words == []:
		print ",".join(nums)
	elif nums == []:
		print ",".join(words)
	else:
		print "|".join([",".join(words), ",".join(nums)])

#test_cases.close()

File: Easy/mixed_content.r
test.cases <- c("8,33,21,0,16,50,37,0,melon,7,apricot,peach,pineapple,17,21",
					"24,13,14,43,41")

#args <- commandArgs(trailingOnly=TRUE)
#test.cases <- strsplit(readLines(args[[1]], warn=FALSE), '\n')
cat(sapply(test.cases, function(test) {
	if (length(test) > 0) {
		words <- strsplit(test, ",")[[1]]
		T <- grep("[[:lower:]]", words, value=T)
		nums <- words[! words %in% T]
		P1 <- paste(T, sep = ",", collapse = ",")
		P2 <- paste(nums, sep = ",", collapse = ",")
		if (length(T) == 0) {
			P2
		}
		else if (length(nums) == 0) {
			P1
		} else {
			paste(c(P1, P2), sep = "|", collapse = "|")
		}
	}
}), sep = "\n")